src/styles/blocks/_global.scss

```diff
@@ -0,0 +1,13 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
```

Comment:
it's better not using any rules besides box-sizing: border-box; at `*` selector

Reply:
How about:
*,
*::after,
*::before { 
}?